### DISCUSSION TITLE
友情链接问题

### DISCUSSION THREAD

**Discussion by anyshift** [Tokinx/Wing#24] [问答] [answered]
在[如何设置友情链接](https://github.com/Tokinx/Wing/wiki/%E5%A6%82%E4%BD%95%E8%AE%BE%E7%BD%AE%E5%8F%8B%E6%83%85%E9%93%BE%E6%8E%A5)页面中，有一处我在我的WP上没找到：

> 在页面中间部位有博主信息板块，其中的设置按钮为友情链接设置选项：
> ![](https://user-images.githubusercontent.com/4526339/167235244-6cfaa86e-5af9-43eb-8a45-d818a41def44.png)

我添加了友情链接页面并让其模板指向友情链接，发布后并没有看到博主头像以及下面的按钮，整个友情链接页面是空白的

**Comment by anyshift** [comment, marked as answer]
好吧，开启评论就有显示了。

**Reply by Tokinx** [reply]
hhh，这个模块我跟comment放一起了，导致如果没开启评论也不会显示这个模块，还不确定有没有改的必要

**Reply by fq176** [reply]
怎么开启评论呢 我也没有
